go-dlv.el

```diff
@@ -37,6 +37,7 @@
 ;; https://github.com/emacs-mirror/emacs/blob/8badbad184c75d6a9b17b72900ca091a1bd11397/lisp/progmodes/gud.el#L1594-1698
 
 (require 'gud)
+(require 'gotest)
```

Comment:
Please add a Package-Requires section at the top so that this package is automatically installed via the package manager.

See here how to do it:
https://github.com/benma/visual-regexp-steroids.el/blob/2a50710dea5be872b31ea56f74b4cd57d6e61461/visual-regexp-steroids.el#L8